Podfile

```diff
@@ -70,6 +73,16 @@ post_install do |installer_representation|
         # Only build active arch
         config.build_settings['ONLY_ACTIVE_ARCH'] = 'YES'
       end
+
+      # ShortcutRecorder sets a weird PRODUCT_BUNDLE_IDENTIFIER
+      # change to ours
+      if target.name == "ShortcutRecorder"
+        xcconfig_path = config.base_configuration_reference.real_path
+        xcconfig = File.read(xcconfig_path)
+        xcconfig_mod = xcconfig.gsub(/com.kulakov.ShortcutRecorder/, "com.sequel-ace.sequel-ace")
+        File.open(xcconfig_path, "w") { |file| file << xcconfig_mod }
+        config.build_settings['PRODUCT_BUNDLE_IDENTIFIER'] = 'com.sequel-ace.sequel-ace'
+      end
```

Comment:
Does it set a weird bundle identifier for Shortcut Recorder or for the root app? If for shortcut recorder, should we maybe set it to com.sequel-ace.sequel-ace.shortcutRecorder (to make bundle ID unique)?

Reply:
for Shortcut Recorder or for the root app, will tale a look at com.sequel-ace.sequel-ace.shortcutRecorder

Reply:
doesn't seem to work, it doesn't create a shortcutRecorder, the assets are in the main bundle.

Reply:
I just wonder what this does for bundle ID stuff in the beta variant of Sequel Ace. The bundle identifier is different for the beta builds. This seems quite screwy of a thing for a pod to do! @Kaspik?